libraries/AP_Camera/AP_Camera_Backend.h

```diff
@@ -183,6 +196,10 @@ class AP_Camera_Backend
     Location last_location;         // Location that last picture was taken at (used for trigg_dist calculation)
     uint16_t image_index;           // number of pictures taken since boot
     bool last_is_armed;             // stores last arm/disarm state. true if it was armed lastly
+    bool _got_camera_info;      // true once camera has provided CAMERA_INFORMATION
```

Comment:
If these are only used for the image tracking then let's put these into a structure